.cbmc-batch/stubs/aws_hash_iter_overrides.c

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Same as before: note that this is a duplicate of c-common, what the change is, and issue to merge them.

Reply:
I created issue https://github.com/aws/aws-encryption-sdk-c/issues/535 to indicate this should be transfer to AWS C Common.